src/views/Viewer.vue

```diff
@@ -713,6 +724,18 @@ export default {
 		onClose() {
 			this.Viewer.onClose()
 		},
+
+		async onDelete() {
+			const url = this.root + this.currentFile.filename
+			await axios.delete(url)
```

Comment:
We need to handle failure.
Please add a try {}...catch wrapper and show a notification on failure (and log in the console :wink: )

Reply:
Done, how to show the error to the user?

Reply:
https://github.com/nextcloud/nextcloud-dialogs/#toasts

Reply:
Done, thanks!